bazel/repository_locations.bzl

```diff
@@ -1376,6 +1376,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2022-03-30",
         cpe = "N/A",
     ),
+    rules_license = dict(
```

Comment:
New internal bazel dep which shows up in the dependency validation queries

Reply:
wow - cool addition - we had started to add something similar - im wondering if/how we can hook  this in

Reply:
We definitely can